FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -369,6 +395,9 @@ get_options()
 	if [ ! -z "${p_conf2hires}" ]; then
 		log_Msg "Include -conf2hires flag in recon-all: ${p_conf2hires}"
 	fi
+	if [ ! -z "${p_pipelinemode}" ] ; then
+  		log_Msg "Set --pipeline-mode to: ${p_pipelinemode}"
```

Comment:
--procesing-mode and p_pipeline_mode

Reply:
Sorry, mistyped that as well.  The proposed lines would be:
```
if [ ! -z "${p_processing_mode}" ] ; then
  		log_Msg "Set --processing-mode to: ${p_processing_mode}"
```

Reply:
fixed.